### ISSUE TITLE
[BUG][WEBJS] - Session could not be started (ERR_CONNECTION_TIMED_OUT at https://web.whatsapp.com/)

### ISSUE THREAD

**Issue by allgerasimof** [allburov/whatsapp-http-api#616]
### Describe the bug

I want to test the service, but the session does not start. Permanent "starting" status.

### Version

```json
{
  "version": "2024.10.10",
  "engine": "WEBLS",
  "tier": "CORE"
}
```

## Steps

**To Reproduce** Steps to reproduce the behavior:

1. Run the command below: docker pull devlikeapro/waha
2. Run WhatsApp HTTP API: docker run -it --rm -p 3000:3000/tcp --name waha devlikeapro/waha
3. Open WAHA Dashboard
4. See **default** session in 'STOPPED' status
5. Click start **default** session
6. Session could not be started

### Expected behavior

Start of session and status "SCAN_QR".

### Requests - Responses

```
root@cv4229311:~# docker pull devlikeapro/waha
Using default tag: latest
latest: Pulling from devlikeapro/waha
ba83bbfca944: Pull complete
48e779000ed2: Pull complete
6d691dff6d17: Pull complete
54cfd3060466: Pull complete
b02a5b714618: Pull complete
95157bce55bf: Pull complete
35daf8b1075a: Pull complete
056622d90eb7: Pull complete
4f4fb700ef54: Pull complete
2f8c6e43683a: Pull complete
6a8e40b38898: Pull complete
7e86b216da4b: Pull complete
4b370c7668f7: Pull complete
767b39a90f57: Pull complete
f176405a877c: Pull complete
5f6d7fe3550d: Pull complete
f09b63133b4e: Pull complete
c8aef233cf39: Pull complete
d80cc16cc525: Pull complete
Digest: sha256:f983718689fffccbd16e437ac2d8a9167961b134ce6d5046ac4ec8ffd3282246
Status: Downloaded newer image for devlikeapro/waha:latest
docker.io/devlikeapro/waha:latest
root@cv4229311:~# docker run -it --rm -p 3000:3000/tcp --name waha devlikeapro/w aha
[07:55:42.705] INFO (Storage/1): Directory '/tmp/whatsapp-files' created from sc ratch
[07:55:43.046] INFO (NestFactory/1): Starting Nest application...
[07:55:43.046] INFO (InstanceLoader/1): PassportModule dependencies initialized
[07:55:43.046] INFO (InstanceLoader/1): ConfigHostModule dependencies initialize d
[07:55:43.046] INFO (InstanceLoader/1): LoggerModule dependencies initialized
[07:55:43.046] INFO (InstanceLoader/1): TerminusModule dependencies initialized
[07:55:43.046] INFO (InstanceLoader/1): ConfigModule dependencies initialized
[07:55:43.046] INFO (InstanceLoader/1): ConfigModule dependencies initialized
[07:55:43.046] INFO (InstanceLoader/1): ServeStaticModule dependencies initializ ed
[07:55:43.046] INFO (InstanceLoader/1): MediaLocalStorageModule dependencies ini tialized
[07:55:43.046] INFO (InstanceLoader/1): ServeStaticModule dependencies initializ ed
[07:55:43.046] INFO (InstanceLoader/1): AppModuleCore dependencies initialized
[07:55:43.046] INFO (RoutesResolver/1): AuthController {/api/:session/auth}:
[07:55:43.046] INFO (RouterExplorer/1): Mapped {/api/:session/auth/qr, GET} rout e
[07:55:43.046] INFO (RouterExplorer/1): Mapped {/api/:session/auth/request-code,  POST} route
[07:55:43.046] INFO (RouterExplorer/1): Mapped {/api/:session/auth/authorize-cod e, POST} route
[07:55:43.046] INFO (RouterExplorer/1): Mapped {/api/:session/auth/captcha, GET}  route
[07:55:43.046] INFO (RouterExplorer/1): Mapped {/api/:session/auth/captcha, POST } route
[07:55:43.046] INFO (RoutesResolver/1): SessionsController {/api/sessions}:
[07:55:43.046] INFO (RouterExplorer/1): Mapped {/api/sessions, GET} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session, GET} rou te
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session/me, GET}  route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session, PUT} rou te
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session, DELETE}  route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session/start, PO ST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session/stop, POS T} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session/logout, P OST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/:session/restart,  POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/start, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/stop, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sessions/logout, POST} rout e
[07:55:43.049] INFO (RoutesResolver/1): ChattingController {/api}:
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendText, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendImage, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendFile, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendVoice, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendVideo, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendButtons, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/forwardMessage, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendSeen, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/startTyping, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/stopTyping, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/reaction, PUT} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/star, PUT} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendPoll, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendLocation, POST} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendLinkPreview, POST} rout e
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/messages, GET} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendText, GET} route
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/sendContactVcard, POST} rou te
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/checkNumberStatus, GET} rou te
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/reply, POST} route
[07:55:43.049] INFO (RoutesResolver/1): ChatsController {/api/:session/chats}:
[07:55:43.049] INFO (RouterExplorer/1): Mapped {/api/:session/chats, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId, DEL ETE} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/mess ages, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/mess ages/:messageId, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/mess ages, DELETE} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/mess ages/:messageId, DELETE} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/mess ages/:messageId, PUT} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/arch ive, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/unar chive, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/chats/:chatId/unre ad, POST} route
[07:55:43.050] INFO (RoutesResolver/1): ChannelsController {/api/:session/channe ls}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels, GET} rou te
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels, POST} ro ute
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels/:id, DELE TE} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels/:id, GET}  route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels/:id/follo w, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels/:id/unfol low, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels/:id/mute,  POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/channels/:id/unmut e, POST} route
[07:55:43.050] INFO (RoutesResolver/1): StatusController {/api/:session/status}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/status/text, POST}  route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/status/image, POST } route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/status/voice, POST } route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/status/video, POST } route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/status/delete, POS T} route
[07:55:43.050] INFO (RoutesResolver/1): LabelsController {/api/:session/labels}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/labels, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/labels/chats/:chat Id, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/labels/chats/:chat Id, PUT} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/labels/:labelId/ch ats, GET} route
[07:55:43.050] INFO (RoutesResolver/1): ContactsController {/api/contacts}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/contacts/all, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/contacts, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/contacts/check-exists, GET}  route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/contacts/about, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/contacts/profile-picture, G ET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/contacts/block, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/contacts/unblock, POST} rou te
[07:55:43.050] INFO (RoutesResolver/1): GroupsController {/api/:session/groups}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups, POST} rout e
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id, GET} r oute
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/setting s/security/info-admin-only, PUT} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/setting s/security/info-admin-only, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/setting s/security/messages-admin-only, PUT} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/setting s/security/messages-admin-only, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id, DELETE } route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/leave,  POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/descrip tion, PUT} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/subject , PUT} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/invite- code, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/invite- code/revoke, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/partici pants, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/partici pants/add, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/partici pants/remove, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/admin/p romote, POST} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/groups/:id/admin/d emote, POST} route
[07:55:43.050] INFO (RoutesResolver/1): PresenceController {/api/:session/presen ce}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/presence, POST} ro ute
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/presence, GET} rou te
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/presence/:chatId,  GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/:session/presence/:chatId/s ubscribe, POST} route
[07:55:43.050] INFO (RoutesResolver/1): ScreenshotController {/api}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/screenshot, GET} route
[07:55:43.050] INFO (RoutesResolver/1): PingController {/ping}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/ping, GET} route
[07:55:43.050] INFO (RoutesResolver/1): HealthController {/health}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/health, GET} route
[07:55:43.050] INFO (RoutesResolver/1): ServerController {/api/server}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/server/version, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/server/environment, GET} ro ute
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/server/status, GET} route
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/server/stop, POST} route
[07:55:43.050] INFO (RoutesResolver/1): ServerDebugController {/api/server/debug }:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/server/debug/heapsnapshot,  GET} route
[07:55:43.050] INFO (RoutesResolver/1): VersionController {/api/version}:
[07:55:43.050] INFO (RouterExplorer/1): Mapped {/api/version, GET} route
[07:55:43.050] INFO (NestApplication/1): Nest application successfully started
[07:55:41.125] INFO (Bootstrap/1): NODE - Catching unhandled rejection enabled
[07:55:41.127] INFO (Bootstrap/1): WAHA (WhatsApp HTTP API) - Running CORE versi on...
[07:55:43.050] INFO (Bootstrap/1): WhatsApp HTTP API is running on: http://[::1] :3000
[07:55:43.050] INFO (Bootstrap/1): Environment {"version":"2024.10.10","engine": "WEBJS","tier":"CORE","browser":"/usr/bin/chromium"}
[07:56:08.036] INFO (1): request completed {"reqId":155,"req":{"id":155,"method" :"GET","url":"/api/version","query":{},"params":{"0":"api/version"}},"res":{"sta tusCode":304},"responseTime":8}
[07:56:08.047] INFO (1): request completed {"reqId":156,"req":{"id":156,"method" :"GET","url":"/api/sessions?all=true","query":{"all":"true"},"params":{"0":"api/ sessions"}},"res":{"statusCode":200},"responseTime":8}
[07:56:08.050] INFO (1): request completed {"reqId":157,"req":{"id":157,"method" :"GET","url":"/api/server/status","query":{},"params":{"0":"api/server/status"}} ,"res":{"statusCode":200},"responseTime":2}
[07:56:16.203] INFO (1): request completed {"reqId":158,"req":{"id":158,"method" :"PUT","url":"/api/sessions/default","query":{},"params":{"0":"api/sessions/defa ult"}},"res":{"statusCode":200},"responseTime":7}
[07:56:17.239] INFO (1): request completed {"reqId":159,"req":{"id":159,"method" :"GET","url":"/api/version","query":{},"params":{"0":"api/version"}},"res":{"sta tusCode":304},"responseTime":2}
[07:56:17.246] INFO (1): request completed {"reqId":160,"req":{"id":160,"method" :"GET","url":"/api/sessions?all=true","query":{"all":"true"},"params":{"0":"api/ sessions"}},"res":{"statusCode":200},"responseTime":1}
[07:56:17.250] INFO (1): request completed {"reqId":161,"req":{"id":161,"method" :"GET","url":"/api/server/status","query":{},"params":{"0":"api/server/status"}} ,"res":{"statusCode":200},"responseTime":3}
[07:56:22.215] INFO (SessionManager/1): 'default' - starting session... {"reqId" :162}
[07:56:22.222] INFO (1): session:default - Session has been started. {"reqId":16 2,"session":"default"}
[07:56:22.223] WARN (SessionManager/1): session:default - Error while getting en gine info {"reqId":162,"session":"default"}
    error: "TypeError: Cannot read properties of null (reading 'evaluate')"
[07:56:22.225] INFO (1): request completed {"reqId":162,"req":{"id":162,"method" :"POST","url":"/api/sessions/default/start","query":{},"params":{"0":"api/sessio ns/default/start"}},"res":{"statusCode":201},"responseTime":11}
[07:56:23.270] INFO (1): request completed {"reqId":163,"req":{"id":163,"method" :"GET","url":"/api/version","query":{},"params":{"0":"api/version"}},"res":{"sta tusCode":304},"responseTime":2}
[07:56:23.275] INFO (1): request completed {"reqId":164,"req":{"id":164,"method" :"GET","url":"/api/sessions?all=true","query":{"all":"true"},"params":{"0":"api/ sessions"}},"res":{"statusCode":200},"responseTime":1}
[07:56:23.279] INFO (1): request completed {"reqId":165,"req":{"id":165,"method" :"GET","url":"/api/server/status","query":{},"params":{"0":"api/server/status"}} ,"res":{"statusCode":200},"responseTime":3}
[25:52:1029/075623.484121:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
[25:52:1029/075623.490097:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
[25:49:1029/075623.516297:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
[25:49:1029/075623.516374:ERROR:bus.cc(407)] Failed to connect to the bus: Could  not parse server address: Unknown address type (examples of valid types are "tc p" and on UNIX "unix")
[25:49:1029/075623.516391:ERROR:bus.cc(407)] Failed to connect to the bus: Could  not parse server address: Unknown address type (examples of valid types are "tc p" and on UNIX "unix")
[25:49:1029/075623.516401:ERROR:bus.cc(407)] Failed to connect to the bus: Could  not parse server address: Unknown address type (examples of valid types are "tc p" and on UNIX "unix")
[25:49:1029/075623.516410:ERROR:bus.cc(407)] Failed to connect to the bus: Could  not parse server address: Unknown address type (examples of valid types are "tc p" and on UNIX "unix")
[25:25:1029/075623.553822:ERROR:system_network_context_manager.cc(854)] Cannot u se V8 Proxy resolver in single process mode.
[25:25:1029/075623.558037:ERROR:policy_logger.cc(156)] :components/enterprise/br owser/controller/chrome_browser_cloud_management_controller.cc(161) Cloud manage ment controller initialization aborted as CBCM is not enabled. Please use the `- -enable-chrome-browser-cloud-management` command line flag to enable it if you a re not using the official Google Chrome build.
[25:49:1029/075623.597195:ERROR:bus.cc(407)] Failed to connect to the bus: Could  not parse server address: Unknown address type (examples of valid types are "tc p" and on UNIX "unix")
[25:49:1029/075623.597236:ERROR:bus.cc(407)] Failed to connect to the bus: Could  not parse server address: Unknown address type (examples of valid types are "tc p" and on UNIX "unix")
[25:25:1029/075623.604853:ERROR:system_network_context_manager.cc(854)] Cannot u se V8 Proxy resolver in single process mode.

DevTools listening on ws://127.0.0.1:45279/devtools/browser/8f164389-d803-4bfe-8 554-d5a6850b6f37
[25:78:1029/075623.870358:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
[25:78:1029/075623.873969:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
[25:78:1029/075623.877738:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
[25:78:1029/075623.879935:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
[25:78:1029/075623.879968:ERROR:bus.cc(407)] Failed to connect to the bus: Faile d to connect to socket /run/dbus/system_bus_socket: No such file or directory
Warning: vkCreateInstance failed with VK_ERROR_INCOMPATIBLE_DRIVER
    at CheckVkSuccessImpl (../../third_party/dawn/src/dawn/native/vulkan/VulkanError.cpp:101)
    at CreateVkInstance (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:493)
    at Initialize (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:379)
    at Create (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:301)
    at operator() (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:556)

Warning: Failed to load libEGL.so
    at DiscoverPhysicalDevices (../../third_party/dawn/src/dawn/native/opengl/BackendGL.cpp:74)

[07:58:34.249] ERROR (WhatsappSession/1): session:default - net::ERR_CONNECTION_TIMED_OUT at https://web.whatsapp.com/ {"reqId":162,"session":"default"}
    err: {
      "type": "Error",
      "message": "net::ERR_CONNECTION_TIMED_OUT at https://web.whatsapp.com/",
      "stack":
          Error: net::ERR_CONNECTION_TIMED_OUT at https://web.whatsapp.com/
              at navigate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/Frame.js:184:27)
              at async Deferred.race (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/util/Deferred.js:36:20)
              at async CdpFrame.goto (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/Frame.js:150:25)
              at async CdpPage.goto (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/api/Page.js:569:20)
              at async WebjsClient.initialize (/app/node_modules/whatsapp-web.js/src/Client.js:328:9)
    }
[07:58:34.251] INFO (WhatsappSession/1): session:default - Job scheduled with timeout 2000 ms {"reqId":162,"session":"default","job":"start-engine","class":"SingleDelayedJobRunner"}
[80:108:1029/075836.496084:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[80:104:1029/075836.498601:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[80:104:1029/075836.498687:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[80:104:1029/075836.498700:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[80:104:1029/075836.498730:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[80:104:1029/075836.498742:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[80:108:1029/075836.501455:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[80:80:1029/075836.508624:ERROR:system_network_context_manager.cc(854)] Cannot use V8 Proxy resolver in single process mode.
[80:80:1029/075836.510432:ERROR:policy_logger.cc(156)] :components/enterprise/browser/controller/chrome_browser_cloud_management_controller.cc(161) Cloud management controller initialization aborted as CBCM is not enabled. Please use the `--enable-chrome-browser-cloud-management` command line flag to enable it if you are not using the official Google Chrome build.
[80:104:1029/075836.526231:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[80:104:1029/075836.526398:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[80:80:1029/075836.533024:ERROR:system_network_context_manager.cc(854)] Cannot use V8 Proxy resolver in single process mode.

DevTools listening on ws://127.0.0.1:35377/devtools/browser/71359472-3d97-4b81-be82-0f79bc5b7905
[80:130:1029/075836.658809:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[80:130:1029/075836.660511:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[80:130:1029/075836.661418:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[80:130:1029/075836.662269:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[80:130:1029/075836.662856:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[07:58:36.253] DEBUG (WhatsappSession/1): session:default - Running job... {"reqId":162,"session":"default","job":"start-engine","class":"SingleDelayedJobRunner"}
[07:58:36.255] DEBUG (WhatsappSession/1): session:default - Browser is ready to be closed: true {"reqId":162,"session":"default"}
[07:58:36.265] DEBUG (WhatsappSession/1): session:default - Job finished {"reqId":162,"session":"default","job":"start-engine","class":"SingleDelayedJobRunner"}
Warning: vkCreateInstance failed with VK_ERROR_INCOMPATIBLE_DRIVER
    at CheckVkSuccessImpl (../../third_party/dawn/src/dawn/native/vulkan/VulkanError.cpp:101)
    at CreateVkInstance (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:493)
    at Initialize (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:379)
    at Create (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:301)
    at operator() (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:556)

Warning: Failed to load libEGL.so
    at DiscoverPhysicalDevices (../../third_party/dawn/src/dawn/native/opengl/BackendGL.cpp:74)

[08:00:47.369] ERROR (WhatsappSession/1): session:default - net::ERR_CONNECTION_TIMED_OUT at https://web.whatsapp.com/ {"reqId":162,"session":"default"}
    err: {
      "type": "Error",
      "message": "net::ERR_CONNECTION_TIMED_OUT at https://web.whatsapp.com/",
      "stack":
          Error: net::ERR_CONNECTION_TIMED_OUT at https://web.whatsapp.com/
              at navigate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/Frame.js:184:27)
              at async Deferred.race (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/util/Deferred.js:36:20)
              at async CdpFrame.goto (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/Frame.js:150:25)
              at async CdpPage.goto (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/api/Page.js:569:20)
              at async WebjsClient.initialize (/app/node_modules/whatsapp-web.js/src/Client.js:328:9)
    }
[08:00:47.370] INFO (WhatsappSession/1): session:default - Job scheduled with timeout 2000 ms {"reqId":162,"session":"default","job":"start-engine","class":"SingleDelayedJobRunner"}
[131:159:1029/080049.497980:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[131:155:1029/080049.500388:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[131:155:1029/080049.500470:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[131:155:1029/080049.500484:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[131:155:1029/080049.500494:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[131:155:1029/080049.500504:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[131:159:1029/080049.505274:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[131:131:1029/080049.511359:ERROR:system_network_context_manager.cc(854)] Cannot use V8 Proxy resolver in single process mode.
[131:131:1029/080049.513456:ERROR:policy_logger.cc(156)] :components/enterprise/browser/controller/chrome_browser_cloud_management_controller.cc(161) Cloud management controller initialization aborted as CBCM is not enabled. Please use the `--enable-chrome-browser-cloud-management` command line flag to enable it if you are not using the official Google Chrome build.
[131:155:1029/080049.528546:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[131:155:1029/080049.528590:ERROR:bus.cc(407)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
[131:131:1029/080049.535684:ERROR:system_network_context_manager.cc(854)] Cannot use V8 Proxy resolver in single process mode.

DevTools listening on ws://127.0.0.1:41541/devtools/browser/852ec7cc-0bd6-4e2c-b937-d05edb53f18c
[131:181:1029/080049.669082:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[131:181:1029/080049.670615:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[131:181:1029/080049.671459:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[131:181:1029/080049.672225:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[131:181:1029/080049.672870:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[08:00:49.372] DEBUG (WhatsappSession/1): session:default - Running job... {"reqId":162,"session":"default","job":"start-engine","class":"SingleDelayedJobRunner"}
[08:00:49.373] DEBUG (WhatsappSession/1): session:default - Browser is ready to be closed: true {"reqId":162,"session":"default"}
[08:00:49.378] DEBUG (WhatsappSession/1): session:default - Job finished {"reqId":162,"session":"default","job":"start-engine","class":"SingleDelayedJobRunner"}
Warning: vkCreateInstance failed with VK_ERROR_INCOMPATIBLE_DRIVER
    at CheckVkSuccessImpl (../../third_party/dawn/src/dawn/native/vulkan/VulkanError.cpp:101)
    at CreateVkInstance (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:493)
    at Initialize (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:379)
    at Create (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:301)
    at operator() (../../third_party/dawn/src/dawn/native/vulkan/BackendVk.cpp:556)

Warning: Failed to load libEGL.so
    at DiscoverPhysicalDevices (../../third_party/dawn/src/dawn/native/opengl/BackendGL.cpp:74)
```

**Comment by devlikepro** [comment]
Make sure you have connection to  https://web.whatsapp.com/

```bash
wget https://web.whatsapp.com/
```

Something with your internet connection.

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
try NOWEB engine

docker run -it -p 3000:3000/tcp -e "WHATSAPP_DEFAULT_ENGINE=NOWEB" devlikeapro/waha

If this one doesn't work as well - 100% something with internet, you can try using proxies
https://waha.devlike.pro/docs/how-to/proxy/

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
